packages/picasso.js/src/core/scales/color/categorical.js

```diff
@@ -12,6 +12,35 @@ const DEFAULT_EXPLICIT_SETTINGS = {
   range: []
 };
 
+/**
+ * Overrides the range with the values specified in the explicit object if possible.
+ * If there are no overrides, the values from explicit object will be injected into
+ * the range in the order specified by the domain.
+ * If the index is out of bounds, no injection or override will be performed.
+ * @param {Array} range - colors
+ * @param {Array} domain - values to be colored
+ * @param {Object} explicit - Object containing explicit domain and range to be overriden or injected.
```

Comment:
add `@ignore`

Reply:
fixed!

Reply:
Sure you've added it? Can't see it @nguyenlam1